test/import/ensureConfigurationFile.test.ts

```diff
@@ -457,202 +388,13 @@ describe('ensureConfigurationFile', () => {
         'Artifact Index': 'artifacts.html',
         Support: { 'FHIR Spec': 'new-tab external http://hl7.org/fhir/R4/index.html' }
       },
-      history: {
-        current: 'http://build.fhir.org/ig/example/example-ig',
-        '0.0.1': {
-          fhirversion: '4.0.1',
-          date: '2099-01-01',
-          desc: 'Initial STU ballot (Mmm yyyy Ballot)',
-          path: 'http://example.org/fhir/STU1',
-          status: 'ballot',
-          sequence: 'STU 1',
-          current: true
-        }
-      },
       indexPageContent: 'Provides a simple example of how FSH can be used to create an IG'
     });
 
     // Template should be commented out since we use the one from ig.ini
     expect(configText).toMatch('# template:');
   });
 
-  it('should generate an appropriate config for a tank w/ package.json and package-list.json', () => {
```

Comment:
I don't know if maybe this is still irrelevant code that should be deleted even _with_ us copying package-list.json, but I'm just calling it out in case you think we should re-add it now that I've asked us to keep copying package-list.json.

Reply:
Thanks for calling this out, but I think this code is still irrelevant. This was code testing for generating a config out of an existing package-list, and we may now have a package-list that exists, but its existence does not impact generation of a config.